src/components/pages/CallPage.jsx

```diff
@@ -45,6 +45,8 @@ export default class CallPage extends React.Component {
                         let tutorialMessage = 'tutorial.notes.targetInfoPhone';
                         if (nextProps.caller.get('has_voip_credentials')) {
                             tutorialMessage = 'tutorial.notes.targetInfoVoip';
+                        } else if (nextProps.caller.get('has_voip_embed')) {
+                            tutorialMessage = 'tutorial.notes.targetInfoVoip';
```

Comment:
This doesn't seem to work, or at least I am not getting this message even though the iframe is there. But IMO it's not a blocker.

Reply:
indeed this needs to be voip_embed_url now, left over wrong variable from an earlier iteration